### PR TITLE
build: update dependencies for 0.3.2 release

### PR DESCRIPTION
**Pull request by rokroskar**

### CODE REVIEW

File: charts/renku/requirements.lock [hunk 1]
```diff
@@ -1,10 +1,10 @@
 dependencies:
 - name: renku-ui
   repository: https://swissdatasciencecenter.github.io/helm-charts/
-  version: 0.4.0
+  version: 0.4.1
 - name: renku-notebooks
   repository: https://swissdatasciencecenter.github.io/helm-charts/
-  version: 0.3.1
+  version: 0.3.2
```
Comment (jirikuncar) [inline charts/renku/requirements.lock]: 👍

File: charts/renku/values.yaml [hunk 2]
```diff
@@ -333,10 +333,10 @@ gitlab:
   password: gitlabadmin
 
   ## Gitlab image
-  # image:
+  image:
   #   pullPolicy: IfNotPresent
-  #   repository: gitlab/gitlab-ce
-  #   tag: latest
+    repository: gitlab/gitlab-ce
+    tag: 11.7.7-ce.0
```
Comment (fgeorgatos) [inline charts/renku/values.yaml]: while at it, why not `11.7.11`?

Comment (rokroskar) [reply charts/renku/values.yaml]: thanks, missed that one

Comment (rokroskar) [reply charts/renku/values.yaml]: done in 3665cfe

File: charts/renku/requirements.lock (hunk 1 above)
Comment (rokroskar) [reply charts/renku/requirements.lock]: ?

File: charts/renku/values.yaml [hunk 3]
```diff
@@ -333,10 +333,10 @@ gitlab:
   password: gitlabadmin
 
   ## Gitlab image
-  # image:
+  image:
   #   pullPolicy: IfNotPresent
-  #   repository: gitlab/gitlab-ce
-  #   tag: latest
+    repository: gitlab/gitlab-ce
+    tag: 11.7.11-ce.0
```
Comment (pameladelgado) [inline charts/renku/values.yaml]: what about using 11.8.7-ce.0 ?

File: charts/renku/values.yaml [hunk 4]
```diff
@@ -51,7 +51,7 @@ global:
     ## Renku version to be used (currently for project creation in
     ## the UI) If not set explicitly the version will be picked up
     ## from the respective renku (sub)chart.
-    version: 0.3.1
+    version: 0.3.2
```
Comment (pameladelgado) [inline charts/renku/values.yaml]: :+1:

File: charts/renku/values.yaml (hunk 3 above)
Comment (rokroskar) [reply charts/renku/values.yaml]: I'd rather not since I just deployed it on dev ~1 hour ago :)

**Pull request merged**